vitest.config.mts

```diff
@@ -0,0 +1,45 @@
+import { resolve, dirname } from 'node:path';
+import { fileURLToPath } from 'node:url';
+import { defineConfig } from 'vitest/config';
+
+const CURRENT_DIR = dirname(fileURLToPath(import.meta.url));
+const WORKSPACE_ROOT = resolve(CURRENT_DIR, './');
+
+// Checking the environment variables simplifies the scripts in the package.json
+// We use `cross-env BROWSER=true vitest` instead of `vitest --project "browser/*"`
+// Which allows us to run `pnpm test:browser --project "x-charts"` for example.
+const getWorkspaces = () => {
+  const getFill = () => {
+    const isJSDOM = process.env.JSDOM === 'true';
```

Comment:
Would it make sense to delete these env vars after this function has run? To avoid contributors from using them instead of feature detection in the tests?

Reply:
Yup, I can delete them.

Just not sure why we would want feature detection in the tests 🤔

Reply:
> Just not sure why we would want feature detection in the tests 🤔

Defensive code, it's portable to any runtime by default, no need to add external signals about how to run it. But this is very much in the "personal preference" and "suggestion" area of code review. Can safely be ignored afaic.

Reply:
done